Builds not running after merge

Please check again

Builds not run after PR "Feature/101 chest implement loot #103"

``` 
Assets/GothicVR/Scripts/Demo/DemoContainerLoot.cs(10,26): error CS0234: The type or namespace name 'Progress' does not exist in the namespace 'UnityEditor' (are you missing an assembly reference?)
```

--- a/Assets/GothicVR/Scripts/Demo/DemoContainerLoot.cs
+++ b/Assets/GothicVR/Scripts/Demo/DemoContainerLoot.cs
@@ -7,7 +7,6 @@ using System;
 using System.Collections.Generic;
 using GVR.Phoenix.Util;
 using UnityEngine;
-using static UnityEditor.Progress;
 
 namespace GVR.Demo
 {

